scylla/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "scylla"
 version = "1.2.0"
 edition = "2021"
-rust-version = "1.81"
+rust-version = "1.82"
 description = "Async CQL driver for Rust, optimized for Scylla, fully compatible with Apache Cassandra™"
 repository = "https://github.com/scylladb/scylla-rust-driver"
 readme = "../README.md"
```

Comment:
Have you left the MSRV on scylla-macros, scylla-cql, and scylla-proxy as 1.81 on purpose? I understand that only scylla needs the bump due to only it using `rustyline`.

Reply:
scylla-cql actually has 1.70. I performed an experiment: updated it and executed clippy to see if it proposes any improvements, but there was nothing, so I don't see the point of updating it.
scylla-macros and scylla-proxy: I forgot about them, but I just tested it the same way as scylla-cql, with the same results.